lte/gateway/configs/service_registry.yml

```diff
@@ -56,7 +56,7 @@ services:
   sgs_service:
     ip_address: 127.0.0.1
     port: 50073
-  sms_orc8r_gw_mme_service:
+  sms_mme:
```

Comment:
nit: let's name this `sms_mme_service` to identify it with the same grpc servicers that run on mme